README.rst

```diff
@@ -47,23 +50,33 @@ Building on Linux with conda-build
 ::
 
     PYVER=<3.6 or 3.7>
+    NUMPYVER=<1.16 or 1.17>
     conda create -n CBLD python=$PYVER conda-build
     source activate CBLD
-    git clone https://github.com/IntelPython/sdc
+    git clone https://github.com/IntelPython/sdc.git
     cd sdc
-    # build Intel SDC
-    conda build --python $PYVER --override-channels -c numba -c conda-forge -c defaults buildscripts/sdc-conda-recipe
+    # Build Numba
+    conda build --python $PYVER --numpy $NUMPYVER --output-folder <path_to_sdc>/numba_build -c numba -c defaults -c intel --override-channels buildscripts/numba-conda-recipe/recipe
+    # Build Intel SDC
+    conda build --python $PYVER --numpy $NUMPYVER -c file://<path_to_sdc>/numba_build -c numba -c defaults -c conda-forge --override-channels buildscripts/sdc-conda-recipe
 
 Building on Linux with setuptools
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ::
 
     PYVER=<3.6 or 3.7>
-    conda create -n SDC -q -y -c numba -c defaults -c conda-forge python=$PYVER numba pandas scipy mpich pyarrow=0.15.1 gcc_linux-64 gxx_linux-64
+    NUMPYVER=<1.16 or 1.17>
+    conda create -n SDC -q -y -c numba -c defaults -c intel -c conda-forge python=$PYVER numpy=$NUMPYVER pandas=0.25.3 scipy pyarrow=0.15.1 gcc_linux-64 gxx_linux-64 tbb-devel llvmlite=0.31.0rc1=py*_0
```

Comment:
llvmlite is a dependency of numba. Why not remove it from the list?

Reply:
We need specific llvmlite version for Numba

Reply:
Also here we build numba via setuptools thus need to install its dependencies

Reply:
When we will update Numba version we also will have to check new dependencies and modify this line. Is there more convenient way to install numba dependencies?

Also, I think we now could use newer version of llvmlite.